Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2688,6 +2688,25 @@ def test_get_context(self):
         finally:
             loop.close()
 
+    def test_proper_refcounts(self):
+        class Break:
+            def __str__(self):
+                raise Exception("break")
```

Comment:
Let's use a more specific exception:
```suggestion
                raise RuntimeError("break")
```